packages/driver/src/cypress/server.js

```diff
@@ -421,6 +422,8 @@ const create = (options = {}) => {
       const { send, open, abort } = XHR.prototype
       const srh = XHR.prototype.setRequestHeader
 
+      const bridgeContentWindowListener = makeContentWindowListener('cypressXhrBridge', contentWindow)
```

Comment:
@brian-mann This is now only creating once per window load, since it's outside the XHR patching